integration-test/src/test/java/org/jboss/pnc/integration_new/endpoint/BuildEndpointTest.java

```diff
@@ -239,7 +239,7 @@ public void shouldGetBuildLogs() throws ClientException {
 
     @Test
     public void shouldFailToGetSshCredentialsForUserThatDidntTrigger() {
-        BuildClient client = new BuildClient(RestClientConfiguration.asUser());
+        BuildClient client = new BuildClient(RestClientConfiguration.asSystem());
```

Comment:
Why did you cahnge this?

Reply:
I had to change authentication to "demo-user" because as old "user" when building inside the code I would have User as null because we don't have user with username:"user" in our demo-data. 

In this case that buildRecord, according to demo-data, was built by demo-user so I had to change the user in request. The "system" was our only other user to choose from.